status-im/status-react
# Disable group chats in dev mode. Closes #4396

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #4396 

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
This PR changes the options list method so that it only adds the group chat button if dev mode is enabled or the `GROUP_CHATS_ENABLED` flag is enabled. It also sets `GROUP_CHATS_ENABLED` to 0 for all configs except for e2e.

### Steps to test:
- Open Status with dev mode disabled
- Click + button
- Verify that group chat option is not available
- Enable development mode
- Click + button
- Verify that group chat option is now available



status: ready
